### PR TITLE
Added marker and line options for `forestplot`

### PR DESCRIPTION
**Pull request by fonnesbeck**
Added options for:

- line width
- color
- marker
- marker size
- font size

Defaults are no longer hard-coded.

### CODE REVIEW

File: pymc3/plots/forestplot.py [hunk 1]
```diff
@@ -101,22 +110,24 @@ def _plot_tree(ax, y, ntiles, show_quartiles):
...
+    ax.errorbar(x=(ntiles[0], ntiles[-1]), y=(y, y), linewidth=int(linewidth/2), color=color)
     return ax
 
 
 def forestplot(trace_obj, varnames=None, transform=identity_transform, alpha=0.05, quartiles=True,
                rhat=True, main=None, xtitle=None, xlim=None, ylabels=None,
-               chain_spacing=0.05, vline=0, gs=None, plot_transformed=False):
+               chain_spacing=0.05, vline=0, gs=None, plot_transformed=False,
+               marker='o', markersize=4, color='blue', linewidth=2,
+               fontsize=None):
```
Comment (twiecki) [inline pymc3/plots/forestplot.py]: Why not pipe through **kwargs?

File: pymc3/plots/forestplot.py [hunk 2]
```diff
@@ -258,6 +250,12 @@ def forestplot(trace_obj, varnames=None, transform=identity_transform, alpha=0.0
 
             # Y coordinate with offset
             y = -var + offset[j]
+            
+            marker = plot_kwargs.pop('marker', 'o')
+            markersize = plot_kwargs.pop('markersize', 4)
+            color = plot_kwargs.pop('color', 'blue')
+            linewidth = plot_kwargs.pop('linewidth', 2)
+            fontsize = plot_kwargs.pop('fontsize', None)
```
Comment (twiecki) [inline pymc3/plots/forestplot.py]: Can't we just forward all of **plot_kwargs to the plotting function below?

Comment (fonnesbeck) [reply pymc3/plots/forestplot.py]: I suppose so. `fontsize` gets used further down, but I can push everything into `_plot_tree`.

**Comment by twiecki** [comment]
LGTM.

**Pull request merged**